development/azure-devops-mock/src/routes/builds.js

```diff
@@ -7,8 +7,7 @@ const builds = [];
 const deploys = [];
 export const buildsRoute = async (req, res) => {
   const params = JSON.parse(req.body.parameters);
-  const isDeploy = !!params.APP_DEPLOY_TOKEN;
-
+  const isDeploy = parseInt(req.body.definition.id) === 81;
```

Comment:
should we have a simple comment about why 81? :)

Reply:
Valid point. Fixed it. No magic variables.